qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -53,39 +55,71 @@ def join_network(self):
     def sc_add_options(self, parser):
         pass
 
-    def sc_generate_genesis_data(self):
-        return None
-
     def sc_setup_chain(self):
-        initialize_sc_chain_clean(self.options.tmpdir, 1)
+        initialize_default_sc_chain_clean(self.options.tmpdir, 1)
 
     """
     Bootstrap some sidechain nodes.
     
     Parameters:
-     - number_of_sidechains_nodes: the number of sidechain nodes to be bootstrapped
-     - number_of_accounts_per_sidechain: an array containing the numbers of accounts to be created for each sidechain node.
-            For example [2, 4, 5] for add 2 accounts to the first node, 4 accounts to the second one and 5 accounts to the third one.
-     - withdrawal_epochs_length:
-     - mainchain_node: the mainchain node
+     - network: a map of:
+        - key: the mainchain node
+        - value: an map of information for each sidechain:
+            - key:
+            - value: an array of:
+                - number_of_accounts: the numbers of accounts to be created
+                - withdrawal_epoch_length
+                - websocket_conf: a websocket configuration. An array of:
+                    - address
+                    - connectionTimeout
+                    - reconnectionDelay
+                    - reconnectionMaxAttempts
+                    
+    Example: 2 mainchain nodes and 3 sidechain nodes (with default websocket configuration) bootstrapped, respectively, from mainchain node first, first, and third.
+    {
+        mc_node_1: {
+            1: [3, 1000, [ws://localhost:8888, 100, 1, 1]]
```

Comment:
Yes, the index should to start from 0.

Reply:
I mean that instead of map with unused keys we can have array of arrays, like
`mc_node_1: [ [3,1000, [ws_data]], [2,1000, ...] ], mc_node_2: [...],...`
Is it right that we don't use that indexes inside?

Reply:
Yes it's tight. The map instead an array of arrays is for a more readability when you want to describe your network.